app/Http/Resources/CommentResource.php

```diff
@@ -0,0 +1,26 @@
+<?php
+
+namespace App\Http\Resources;
+
+use Illuminate\Http\Resources\Json\JsonResource;
+
+class CommentResource extends JsonResource
+{
+    /**
+     * Transform the resource into an array.
+     *
+     * @param  \Illuminate\Http\Request  $request
+     * @return array
+     */
+    public function toArray($request)
+    {
+        return [
+            'id' => $this->id,
+            'user_id' => $this->user_id,
```

Comment:
Вот тут лучше прицепить ресурс юзера, вместо простой айдишки
```suggestion
            'user' => new UserResource($this->user),
```